action.c

```diff
@@ -1430,6 +1430,14 @@ actionWriteErrorFile(action_t *__restrict__ const pThis, const rsRetVal ret,
 				pThis->pszName, pThis->pszErrFile);
 			goto done;
 		}
+		if (pThis->maxErrFileSize > 0) {
+			struct stat statbuf;
+			if (fstat(pThis->fdErrFile, &statbuf) == -1) {
+				LogError(errno, RS_RET_ERR, "failed to fstat %s", pThis->pszErrFile);
+				goto done;
+			}
+			pThis->errFileWritten += statbuf.st_size;
```

Comment:
IMHO adding to the current error file size is incorrect, at least misleading. Shouldn't it plainly set the size?

I guess it works, as the fd is only opened once, but in respect to ability to understand the code this should be changed to a pure assignment. Or am I overlooking something?

Reply:
Thx for the swift reply!
well - if you fstat() the file, isn't that the "current size" - why add it to the previous "current size"? That's my question.

Reply:
Sorry, I didn't get your point. Yes, it is not necessary to add, just assignment.